ibtk/src/lagrangian/FEProjector.cpp

```diff
@@ -124,7 +124,7 @@ qrule_is_nodal(const FEType& fe_type, const QBase* const qrule)
     if (fe_type.family == LAGRANGE || fe_type.family == L2_LAGRANGE || fe_type.family == MONOMIAL)
     {
         nodal_cond = true;
-        if (fe_type.family == MONOMIAL && fe_order != CONSTANT) return false;
+        if (fe_type.family == MONOMIAL && !(fe_order == CONSTANT || fe_order == FIRST)) return false;
```

Comment:
I thought the first-degree monomials were `1, x, y, z`: that doesn't correspond to a nodal quadrature rule. What's the explanation for this change?

Reply:
This check was throwing a `TBOX_ERROR` when building the L2 projection matrix. I guess part of my confusion was that if you choose `FIRST` order L`2_LAGRANGE` then this check returned true for nodal quadrature rule, but `FIRST `order `MONOMIAL` was returning false. Is that what expect? Maybe I'm not understanding why the two situations would be different in terms of using nodal quadrature use.